src/components/Navigation.js

```diff
@@ -0,0 +1,84 @@
+import { NavLink } from 'react-router-dom';
+import { useState } from 'react';
+import MenuIcon from '@mui/icons-material/Menu';
+import CloseIcon from '@mui/icons-material/Close';
+import logo from '../images/logo.png';
+import fb from '../images/fb.svg';
+import google from '../images/google.svg';
+import x from '../images/x.svg';
+import github from '../images/github.svg';
+import linkedin from '../images/linkedin.svg';
+
+const Navigation = () => {
+  const [showMenu, setShowMenu] = useState(false);
+
+  const toggleMenu = () => {
+    setShowMenu(!showMenu);
+  };
+
+  const closeMenu = () => {
+    setShowMenu(false);
+  };
+  return (
+    <>
+      <button
+        type="button"
+        className="md:hidden absolute top-4 left-4 mr-4 p-2 text-gray-500 hover:text-gray-700 focus:outline-none focus:ring focus:ring-gray-200"
+        onClick={toggleMenu}
+      >
+        {showMenu ? <CloseIcon /> : <MenuIcon />}
+      </button>
+      <div className={`${showMenu ? 'flex absolute top-0 left-0 z-50' : 'hidden md:flex'} flex-col pt-4 pb-12 md:pb-1 justify-between items-center h-[100dvh] bg-white  w-full drop-shadow-md`}>
+        <div>
+          <img src={logo} alt="" />
+          <button
+            type="button"
+            className="md:hidden absolute top-4 left-4 mr-4 p-2 text-gray-500 hover:text-gray-700 focus:outline-none focus:ring focus:ring-gray-200"
+            onClick={toggleMenu}
+          >
```

Comment:
- [ ] [OPTIONAL]: :+1:  Great job implementing the mobile version styling! Your work is awe-inspiring. I have a suggestion regarding styling that could make your `navbar` even more stunning. Currently, the logo overlaps with the close menu `button`. You can fix this by increasing the `padding` to `2.5rem` or `3rem`.

![image](https://github.com/Harrynoqs/Final_capstone_frontend/assets/101400298/fd312b4a-5ee1-45b5-a845-72bd37189ca4)